Graphics/GraphicsEngineOpenGL/src/RenderDeviceGLImpl.cpp

```diff
@@ -1253,7 +1253,7 @@ void RenderDeviceGLImpl::FlagSupportedTexFormats()
     FlagFormat(TEX_FORMAT_R32G8X24_TYPELESS,          true                                      );
     FlagFormat(TEX_FORMAT_D32_FLOAT_S8X24_UINT,       true,         BIND_DEPTH_STENCIL          );
     FlagFormat(TEX_FORMAT_R32_FLOAT_X8X24_TYPELESS,   true,         TexBindFlags,     bDekstopGL);
-    FlagFormat(TEX_FORMAT_X32_TYPELESS_G8X24_UINT,    false                                     );
+    FlagFormat(TEX_FORMAT_X32_TYPELESS_G8X24_UINT,    true,         BIND_SHADER_RESOURCE,  false);
```

Comment:
This needs to correctly check for support.
The `GL_DEPTH_STENCIL_TEXTURE_MODE` is supported in GL4.3+ and GLES3.1+